js/components/header.tsx

```diff
@@ -1,10 +1,32 @@
+import { paths } from "../paths";
+import { className } from "../util/dom";
+import { Link, useLocation } from "react-router-dom";
+
 export const Header = () => {
+  const currentLocation = useLocation();
+  const menuSelected =
+    currentLocation.pathname === paths.menu ||
+    currentLocation.pathname === paths.help;
+
   return (
     <div className="w-full bg-gray-200 p-2 flex justify-between">
-      <img src="/images/logo.svg" alt="MBTA" className="w-8" />
-      <a href="/logout" className="underline p-1">
-        Log out
-      </a>
+      <Link to={"/"}>
```

Comment:
**suggestion:** is there some reason this can't just be
```suggestion
      <Link to={paths.root}>
```
now?

Reply:
nope! a423717